### PR TITLE
Emp rework bugfix (fringe case where units can still take mass AoE emp damage)

### PR DESCRIPTION
**Pull request by TheSilverHornet**
Fixed misplaced code

### CODE REVIEW

File: luarules/gadgets/unit_paralyze_damage_limit.lua [hunk 1]
```diff
@@ -47,8 +47,9 @@ for udid, ud in pairs(UnitDefs) do
...
 	
 	if modOptions.emprework==true then
-		unitOhms[udid] = ud.customParams.paralyzemultiplier or 1
-		--Spring.Echo('ohm', ud.customParams.paralyzemultiplier)
+		unitOhms[udid] = ud.customParams.paralyzemultiplier == nil and 1 or tonumber(ud.customParams.paralyzemultiplier)--it arrives as a string because WHY NOT
+		
+		--Spring.Echo('ohm', ud.customParams.paralyzemultiplier, unitOhms[udid])
 		if tonumber(ud.customParams.paralyzemultiplier) or 0 > 0 then
 		--Spring.Echo('ohm', ud.name, ud.customParams.paralyzemultiplier)
 		end
```
Comment (thehobojoe) [inline luarules/gadgets/unit_paralyze_damage_limit.lua:55]: ✂️

File: luarules/gadgets/unit_paralyze_damage_limit.lua [hunk 2]
```diff
@@ -97,16 +98,20 @@ function gadget:UnitPreDamaged(uID, uDefID, uTeam, damage, paralyzer, weaponID,
 					--ohm = ohm * 1.4 -- with 1.4, an EMP resistance of say 1.2 gains EMP only slightly faster but has nearly double max charge capacity relative to lesser units
 				--end
 				thismaxtime = weaponParalyzeDamageTime[weaponID] * ((ohm == 1 and 0.85) or ohm)
+
+
+				if ohm>0 then
+					bufferdamage = hp / 200
+					--rootdamage = (damage / 50) * hp^0.5
+					--Spring.Echo('h damage rootdamage',hp,damage, rootdamage)
```
Comment (thehobojoe) [inline luarules/gadgets/unit_paralyze_damage_limit.lua:106]: ✂️

**Review by thehobojoe — CHANGES_REQUESTED** [review]
Friends don't let friends commit commented code 🙂

File: luarules/gadgets/unit_paralyze_damage_limit.lua (hunk 2 above)
Comment (TheSilverHornet) [reply luarules/gadgets/unit_paralyze_damage_limit.lua:105]: potentially still needed

File: luarules/gadgets/unit_paralyze_damage_limit.lua (hunk 1 above)
Comment (TheSilverHornet) [reply luarules/gadgets/unit_paralyze_damage_limit.lua]: conditional debugging, but removed - can't promise it won't be back next time I'm chasing bugs around

**Pull request merged**